circle.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+
+jobs:
+  version: 2
+
+jobs:
+  linux-performance:
+    docker:
+      - image: swift:5.0-bionic
+    steps:
+      - checkout
+      - run:
+          name: swift test
+          command: swift test -c release --filter "RoutingKitTests.RouterPerformanceTests"
```

Comment:
we should be able to do `swift test --enable-test-discovery` here if you do `swift:5.1-bionic`

Reply:
Will update